appveyor.yml

```diff
@@ -0,0 +1,50 @@
+build: false
+
+environment:
+  matrix:
+    - PYTHON_VERSION: 2.7
+      PYTHON_SVER: 27
+      MINICONDA: C:\Miniconda
+    - PYTHON_VERSION: 3.4
```

Comment:
The limiting factor is the pygraphviz binary - I haven't been able to find one for 3.5 or 3.6 on win32, so I had to go with Chris Gohlke's 3.4 build.